### ISSUE TITLE
Support for decimal alignment in specified columns of a table.

### ISSUE THREAD

**Issue by phelps-sg** [jgm/pandoc#5998]
For tables containing numeric data, It would be very useful to be able to specify that particularly columns of a markdown table should be aligned on decimal points.  

The [MultiMarkDown project](https://github.com/fletcher/MultiMarkdown) has [a syntax for this](https://github.com/fletcher/MultiMarkdown/wiki/MultiMarkdown-Syntax-Guide#tables).  

It is possible to [generate html tables with decimal alignment](https://stackoverflow.com/questions/1363239/aligning-decimal-points-in-html).

**Comment by mb21** [comment]
I guess this would require a change to `data Alignment` in the pandoc document AST... https://github.com/jgm/pandoc-types/blob/master/Text/Pandoc/Definition.hs#L166

**Comment by jgm** [comment]
Yes, it would require an AST change.  Does LaTeX have a syntax for decimal alignment in tables?  Word?

**Comment by mgajda** [comment]
StackOverflow answer indicates that one can `\usepackage{siunitx}` and then `S` table alignment option:
https://tex.stackexchange.com/questions/298279/dot-aligned-table-column-with-defined-width
